src/visuallyCompleteCalculator.ts

```diff
@@ -94,6 +93,11 @@ class VisuallyCompleteCalculator {
     // - wait for simultaneous network and CPU idle
     const didNetworkTimeOut = await new Promise<boolean>(requestAllIdleCallback);
 
+    // if this isn't the most recent navigation, abort
+    if (navigationIndex !== this.navigationCount) {
```

Comment:
I had to think about this for a sec, but I think it makes sense. Do I have the right sequence of events here?

1. `start()` is called for the first time (let's refer to the execution context as `a`)
2.  `a` executes line 73 which increments `this.navigationCount` to 1 and `navigationIndex` gets assigned a value of 1
3. `start()` is called again (`b`)
4. `b` executes line 73 which increments `this.navigationCount` to 2
5. `a` reaches line 97 which resolves to `true` because `navigationIndex` is `1` but `this.navigationCount` is 2
6. `a` is cancelled
7. `b` continues

So we may have multiple execution contexts running and evaluating TTVC, but when it comes time to report the TTVC value for the current navigation, we only do that once (specifically the most recent call) and all other duplicate calls will get cancelled.

Reply:
Yess, you got it exactly!

If you can think of any ideas for how to express the logic more clearly, I'm all ears.  I rewrote this a few times, but it always seems to come out a little more confusingly than I want.